README.md

```diff
@@ -349,3 +349,47 @@ To generate an authentication token run RAILS_ENV=production bin/rails generate_
 ```
 {"invoked_by" => "workflow-service"}
 ```
+
+## File layout
+Objects created prior to August 2024 using the legacy, unversioned layout. Objects created thereafter have the new versioned layout.
```

Comment:
Is something missing from this first sentence?

Reply:
Missing proper grammar? Fixing.